### DISCUSSION TITLE
Download Link for db_resnet50_rotation

### DISCUSSION THREAD

**Discussion by ffalkenberg** [mindee/doctr#1371] [Q&A] [answered]
Hello everyone,

I've been trying to download the `db_resnet50_rotation` model manually from the provided link, but I end up getting `classif_mobilenet_v3_small-24f8ff57.pt` with a file size of 5.90 MB (6,193,331 Bytes). This doesn't seem right to me. Can someone confirm if this is the correct file or guide me to the correct link for downloading `db_resnet50_rotation`?

Thanks!

**Comment by felixdittrich92** [comment, marked as answer]
:)

https://github.com/mindee/doctr/blob/bd5b9959bd0151fa4c03506f985f8665b4bb2f51/doctr/models/detection/differentiable_binarization/pytorch.py#L48
